Contract7/LaGrandeJatteRain/main.py

```diff
@@ -0,0 +1,45 @@
+import random
+import pygame, sys
+from pygame.locals import *
+
+pygame.init()
+clock = pygame.time.Clock()
+
+ball_pos_x = []
+ball_pos_y = []
+ball_speed_x = []
+ball_speed_y = []
+ball_radius = []
+xPos = 0
+yPos = 0
+WIDTH = 750
+HEIGHT = 501
+ball_index = 0
+window = pygame.display.set_mode((WIDTH,HEIGHT),0,32)
+BLACK = (0,0,0)
+BLUE = (0,0,255,6)
+bg = pygame.image.load("Images/BaseImage.jpg")
+num_balls = 9
+num_balls -= 1
+

```

Comment:
Comments? Also spaceing out code into different sections, would help me to understand which bits of code interact with others. For example seperate ball properties from the width and height, as they arn't directly related.